tests/nist/test_worker.py

```diff
@@ -79,7 +79,7 @@ def run_test(test, scanner):
         sys.stderr.write("Unable to run the test! Check the catalog.xml.\n")
         return None
     print(command)
-    with open(test_id + ".stdout", "w") as stdout_file:
+    with open(os.path.join(output_dir, test_id + ".stdout"), "w") as stdout_file:
```

Comment:
As discussed with @mpreisler , it makes sense for the tool not to create the output directory.
Mistakes in the parameters don't end up creating a directory..